### PR TITLE
Hello World

### PR DESCRIPTION
**Pull request by Tuglo**
Linter(A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Read More about linter((https://en.wikipedia.org/wiki/Lint_(software))).)

Html(HTML is an acronym which stands for Hyper Text Markup Language which is used for creating web pages and web applications. Let's see what is meant by Hypertext Markup Language, and Web page)

Css(What is CSS? Cscading Style Sheets, fondly referred to as CSS, is a simple design language intended to simplify the process of making web pages presentable. CSS handles the look and feel part of a web page.)

### CODE REVIEW

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Tuglo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
The project is running successfully ✔️

## Required Changes ♻️

- [ ] Linters should be set correctly for this repository 
- [ ] Your branch name should be descritive eg;  (e.g. 'created_sidebar' or 'added_comments_section')
- [ ] In Your readme file in the technology used section, you should just list the technology you've used eg: `HTML`,  `CSSS`.  and also remove the demo link, for you don't have dome link now 
image

- [ ] Your pull request description should describe things that you've worked on in you branch 
- [ ] Commit messages should be meaningful.

## Optional suggestions
- [ OPTIOANL]  You should not merge master branch in any other branch, Your created branch should be the one to be merged in master, take master as a parent branch 
- [ OPTIONAL]  Try to fix the conflict issue so that it won't affect you at time of merging your branch, when you get approved. 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tuglo** [comment]
Hi zachee ishimwe
I have completed what you are asking for.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+      <h1>Hello Microverse!</h1>
+    </body>
+  </html>
```
Comment (codecaiine) [inline index.html:9]: - [ ] [OPTIONAL]  I recommend you to add it cause The <title> tag is required in HTML documents! for some reasons like defining a title in the browser toolbar or providing a title for the page when it is added to favorites or displays a title for the page in search-engine results

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+      <h1>Hello Microverse!</h1>
+    </body>
+  </html>
+
+
+
+
```
Comment (codecaiine) [inline index.html:13]: - [ ] [OPTIONAL] I also recommend you to remove this empty code to make your code easy to read

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Tuglo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
The project is running successfully :heavy_check_mark:

 ## Required Changes ♻️
 - [ ] For this project Linters should be set correctly for this repository, please make sure that  [linter](https://github.com/microverseinc/linters-config/tree/master/html-css) is set up correctly.
 - [ ]  Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
 - [ ] [OPTIONAL] Your pull Request need to be made from the `feature branch` into the main branch.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @Tuglo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a closed pull request(PR). Also please don't merge your pull request unless it has been approved. Good luck :)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**